core/src/http_advertiser.cpp

```diff
@@ -0,0 +1,69 @@
+#include "include/http_advertiser.h"
+
+HTTPAdvertiser::HTTPAdvertiser()
+{
+    m_manager = new QNetworkAccessManager();
+    connect(m_manager, &QNetworkAccessManager::finished,
+            this, &HTTPAdvertiser::msRequestFinished);
+}
+
+void HTTPAdvertiser::msAdvertiseServer()
+{
+    if (m_masterserver.isValid()) {
+
+        QNetworkRequest request((QUrl (m_masterserver)));
```

Comment:
More brackets than when a 4channer thinks they're cleverly calling out Jewish people on Twitter.

```suggestion
        QNetworkRequest request(QUrl(m_masterserver));
```